packages/data-transformation/csv-to-metrics/entrypoint.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+INPUT_DATABASE="${DBT_TUVA_CI_DATABASE}"
+INPUT_SCHEMA="${DBT_SNOWFLAKE_CI_SCHEMA}"
+dbt build --vars "{"input_database": "$INPUT_DATABASE", "input_schema": "$INPUT_SCHEMA"}"
```

Shellcheck (0.10.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)